components/Nav.tsx

```diff
@@ -25,10 +29,17 @@ const Nav: React.FC = (): React.JSX.Element => {
           GPRG
         </h1>
       </section>
-      <ul className="flex justify-between gap-3 md:gap-4 lg:gap-6 text-[0.75rem] md:text-[1rem]">
+      <span className="md:hidden flex items-center justify-center">
+          <Hamburger 
+            toggled={navbar}
+            toggle={() => setNavbar(!navbar)}
+            duration={0.5}/>
+      </span>
+      </div>
+      <ul className= {`pl-[2px] md:flex justify-between gap-3 md:gap-4 lg:gap-6 text-[1rem] ${navbar? "block":"hidden"}`}>
```

Comment:
* Please change the "px" unit of pl-[2px] to rem or em. px is not that good for responsiveness.
* You can use the Prettier extension to format better.
* Also can you please explain the use of padding-left of 2px here?
* Also change from text-[1rem] to text-base. It's exactly the same but keeps the consistency of the util classes.

Reply:
The pl-[0.15rem] give some space from left in the navbar Element. As the CSS looks better when the hamburger menu is used.
I have installed Prettier extension but don't know why, it didn't worked.

Reply:
Okay understandable. I believe it's pl-[0.125rem] for 2px. You can try installing Tailwind Intellisense extension.

Reply:
Also please send screenshots of the navbar in mobile view both open and close.